tests/fake_texture.rs

```diff
@@ -0,0 +1,18 @@
+extern crate graphics;
+
+use self::graphics::ImageSize;
+
+pub struct FakeTexture {
```

Comment:
This seemed like the easiest way to get an "empty" texture. Maybe there is a way to do it with the existing library?